README.md

```diff
@@ -20,8 +20,19 @@ This project processes clinical data from https://clinicaltrials.gov/, specifica
 ## Set up
 
 ### For Windows Beginners
-
-*To be written*
+1)Open Windows PowerShell
+2)Type these commands includeing any spaces but not includeing brackets
+```
```

Comment:
Use the appropriate language to obtain correct highlighting ( https://github.com/jincheng9/markdown_supported_languages?tab=readme-ov-file#heres-a-full-list-of-supported-languages ).